virtual_machines.go

```diff
@@ -29,14 +29,19 @@ func (vm *VirtualMachine) Get(ctx context.Context, client *Client) error {
 		return err
 	}
 
+	payload, err := client.Payload(&ListVirtualMachines{})
+	if err != nil {
+		return err
+	}
+
 	count := len(vms)
 	if count == 0 {
 		return &ErrorResponse{
 			ErrorCode: ParamError,
-			ErrorText: fmt.Sprintf("missing VirtualMachine. id: %s, name: %s", vm.ID, vm.Name),
+			ErrorText: fmt.Sprintf("missing VirtualMachine. id: %s, name: %s %s", vm.ID, vm.Name, payload),
 		}
 	} else if count > 1 {
-		return fmt.Errorf("more than one VirtualMachine was found. Query: id: %s, name: %s", vm.ID, vm.Name)
+		return fmt.Errorf("more than one VirtualMachine was found. Query: id: %s, name: %s %s", vm.ID, vm.Name, payload)
```

Comment:
As `id` and `name` are not always relevant, we may only show the payload.
`"missing VirtualMachine. query: %s", payload`

Reply:
once we have that, is there a way to refactor the `Get` call?